### DISCUSSION TITLE
Is the Accept header value, "application/graphql-response+json" supported?

### DISCUSSION THREAD

**Discussion by ddoongoor** [Netflix/dgs-framework#1728] [Q&A]
If I send a request with this Accept header, I receive a HTTP 406 Not Accepted response.

**Comment by martinbonnin** [comment]
@ddoongoor Is that still the case? I believe Spring GraphQL supports this now: https://docs.spring.io/spring-graphql/reference/transports.html

**Reply by ddoongoor** [reply]
I haven't checked in a while, since I implemented it myself. When I have some cycles I will check it out. Feel free to close the ticket, if necessary.
